docs/pages/blog/matheus-wichman-joining.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Matheus Wichman joins Material-UI
+description: We are excited to share that Matheus Wichman has joined Material-UI.
+date: 2021-04-05T00:00:00.000Z
+authors: ['oliviertassinari']
+card: true
+---
+
+We are excited to share that [Matheus Wichman](https://github.com/m4theushw) has joined Material-UI.
+He has started a couple of days ago full-time and is now part of the advanced components team.
+
+Before joining Material-UI, Matheus worked as a software engineer at a consulting agency in Brazil.
+He has spent the last three years focusing on React and the frontend world.
+His last project was for a financial institution.
+Matheus is passionate about software development, so much that he started his career very early on.
+
+I was impressed by his capability to quickly jump into an unknown codebase and reverse entropy.
```

Comment:
```suggestion
We were impressed by his capability to quickly jump into an unknown codebase and reverse entropy.
```

"reverse entropy" please translate

Reply:
As opposed to solving one issue on the surface and creating 3 others.

Reply:
```suggestion
We were impressed by his capability to quickly jump into an unknown codebase and solve problems at their root.
```